Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/settings/SettingsFragmentPresenter.java

```diff
@@ -253,6 +253,7 @@ private void addGraphicsSettings(ArrayList<SettingsItem> sl)
 		}
 		else
 		{
+			mSettings.get(SettingsFile.SETTINGS_DOLPHIN).put(SettingsFile.SECTION_CORE, new SettingSection(SettingsFile.SECTION_CORE));
```

Comment:
shouldn't this be within `mSettings.get(SettingsFile.SETTINGS_DOLPHIN).isEmpty()`